src/xhr.js

```diff
@@ -52,7 +52,10 @@ const callbackWrapper = function(request, error, response, callback) {
     error = new Error('XHR Failed with a response of: ' +
                       (request && (reqResponse || request.responseText)));
   }
-
+  // If we have a response hook, call it now.
+  if (beforeResponse && typeof beforeResponse === 'function') {
+    beforeResponse(response);
```

Comment:
I think this function should receive the arguments `error, response` for [parity with the XHR module](https://github.com/videojs/xhr/blob/main/README.md#xhrcallback).